publisher-html/src/main/resources/templates/events/event.hbs

```diff
@@ -1,39 +1,50 @@
 <div class="event">
-        <div class="event-image {{#if event.category}}event-image-{{event.category}}{{/if}}">
-            {{#if event.pictureUrl}}
-                <img src="/picture?url={{event.pictureUrl}}" style="height:100px;max-height:250px;max-width:300px;" />
-            {{else}}
-                <svg title="Event Bild" viewBox="0 0 512 512" height="100px" >
-                    <use xlink:href="#{{#if event.category}}{{event.category}}{{else}}image{{/if}}"></use>
+    <div class="event-image {{#if event.category}}event-image-{{event.category}}{{/if}}">
+        {{#if event.pictureUrl}}
+            <img src="/picture?url={{event.pictureUrl}}" style="height:100px;max-height:250px;max-width:300px;" />
+        {{else}}
+            <svg title="Event Bild" viewBox="0 0 512 512" height="100px">
+                <use xlink:href="#{{#if event.category}}{{event.category}}{{else}}image{{/if}}"></use>
+            </svg>
+        {{/if}}
+        {{#if event.url}}
+            <a class="anchor-to-eventpage" href="{{event.url}}" target="_blank"
+               aria-describedby="Zur Eventseite von {{event.name}} (neues Fenster wird geöffnet)">
+                Zur Eventseite
+            </a>
+        {{/if}}
+
+        {{#or event.accessibleEntry event.accessibleSeats event.accessibleToilets }}
+            <div class="accessibility-details">
+                <svg height="24px" width="24px" xmlns="http://www.w3.org/2000/svg" id="Layer_1" data-name="Layer 1" viewBox="0 0 122.88 122.88"><title>
+                    accessibility</title>
```

Comment:
[nit] https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title should describe objects by its visual appearance, in this case I would suggest to add a descriptive way of what the vector graphic shows e.g. "schwarze Silhouette einer Person"